### DISCUSSION TITLE
80090308: LdapErr: DSID-0C09044E, comment: AcceptSecurityContext error, data 52e, v2580

### DISCUSSION THREAD

**Discussion by raghu431** [dsbenghe/Novell.Directory.Ldap.NETStandard#197] [General]
80090308: LdapErr: DSID-0C09044E, comment: AcceptSecurityContext error, data 52e, v2580

Hi ,
Trying to LDAP with below code in DotNet 6 application.
   con.Connect("ad.xyz.com", "389");
                   con.SecureSocketLayer = true;
                    con.Bind(LdapConnection.LdapV3, "username\\domain", config.Password);

i am getting ""Invalid Credentials" error and but the username and password is correct.

Inner exception:80090308: LdapErr: DSID-0C09044E, comment: AcceptSecurityContext error, data 52e, v2580

**Comment by dsbenghe** [comment]
This is not really an issue - just incorrect usage

**Reply by raghu431** [reply]
.what is the correct usage here and can you provide sample .
Note:
Same username and password format working Directory.Services nuget (Legacy DotNet)
We have two AD domains and same code is working for  one Domain another one its throwing error Inner exception:80090308: LdapErr: DSID-0C09044E, comment: AcceptSecurityContext error, owing error

**Reply by Rose-bin** [reply]
Hi, raghu431,
we have the same problem as described. Did you get your Problem solved? Unfortunately this conversation stopped with your question.
